README.md

```diff
@@ -23,18 +25,23 @@ $ PKG_TYPES=deb PKG_DEB_RELEASES=debian11 docker buildx bake pkg
 # create packages for windows/amd64 platform and output to ./bin folder
 $ docker buildx bake --set *.platform=windows/amd64 --set *.output=./bin pkg
 
-# create packages for all supported platforms and push to crazymax/buildx-packaging:v0.8.1
-$ docker buildx bake --set *.output=type=image,push=true --set *.tags=crazymax/buildx-packaging:v0.8.1 pkg-cross
+# create packages for all supported platforms and push to docker/buildx-pkg:v0.8.1
```

Comment:
Wondering if it should be pushing to `dockereng/` instead of `docker/` (I know previously we used `dockereng/` for more "internal" (not so interesting to the public) images, but not sure if we still use that distinction consistently)

Reply:
Ah yes true, wdyt @ccrone?

Reply:
Changed to `dockereng/buildx-pkg` as it seems the most appropriate location.